packages/fether-react/src/stores/parityStore.js

```diff
@@ -18,10 +20,9 @@ const electron = isElectron() ? window.require('electron') : null;
 const LS_KEY = `${LS_PREFIX}::secureToken`;
 
 export class ParityStore {
-  @observable
-  downloadProgress = 0;
   @observable
   isApiConnected = false;
+  isApiConnected$ = timer(0, 1000).pipe(map(_ => this.isApiConnected));
```

Comment:
Since there is already a timer, I would remove the setInterval below, and do `isApiConnected$.subscribe(this.setApiIsConnected)`, and replace here by `=> this.api.isConnected`